torch/csrc/jit/script/compiler.cpp

```diff
@@ -324,9 +324,21 @@ void liftConstantAttributes(const FunctionSchema& schema, Node* node) {
   JIT_ASSERT(!node->hasAttributes());
   std::vector<Value*> new_inputs;
   Attributes<Node> attributes;
-  for(size_t i = 0; i < node->inputs().size(); ++i) {
+  for(size_t i = 0, n = 0; i < schema.arguments.size(); ++i) {
     const auto& arg = schema.arguments[i];
-    auto input = node->input(i);
+    // this was a builtin with a vararg list lowered,
+    if(arg.type->kind() == TypeKind::ListType) {
+      // we do not support constant lifting of the arg itself
+      if(arg.attribute_info)
```

Comment:
What's attribute info in this case? It's an untyped thing (at least at compile time), so it would deserve a comment.

Reply:
If this input can be a graph attribute rather than an input, attribute_info contains the kind of the graph attribute that this should turn into.  function_schema.h has a comment documenting the format.